src/libcrun/criu.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * crun - OCI runtime written in C
+ *
+ * Copyright (C) 2020 Adrian Reber <areber@redhat.com>
+ * crun is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * crun is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with crun.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#define _GNU_SOURCE
+
+#include <config.h>
+
+#ifdef HAVE_CRIU
+
+#include <unistd.h>
+#include <sys/types.h>
+#include <criu/criu.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include "container.h"
+#include "error.h"
```

Comment:
`make syntax-check` is failing since `error.h` is included but the rest of the file doesn't use the `error` function.

Reply:
also fixed